app/src/webtrigger/types.ts

```diff
@@ -42,9 +42,9 @@ export interface JenkinsEvent extends JenkinsRequest {
 export interface JenkinsPluginConfigEvent extends JenkinsEvent, JenkinsPluginConfig {
 	lastUpdatedOn: Date,
 	ipAddress: string,
-	autoBuildEnabled?: string,
+	autoBuildEnabled?: boolean,
```

Comment:
Looks like we've got duplicate types. Added an item to my todo checklist to clean this up later.